website/src/css/custom.css

```
@@ -6,25 +6,39 @@
 
 /* You can override the default Infima variables here. */
 :root {
-  --ifm-color-primary: #2e8555;
-  --ifm-color-primary-dark: #29784c;
-  --ifm-color-primary-darker: #277148;
-  --ifm-color-primary-darkest: #205d3b;
-  --ifm-color-primary-light: #33925d;
-  --ifm-color-primary-lighter: #359962;
-  --ifm-color-primary-lightest: #3cad6e;
+  --ifm-color-primary: #ff5760;
+  --ifm-color-primary-dark: #d23c43;
+  --ifm-color-primary-darker: #ae262d;
+  --ifm-color-primary-darkest: #540c0f;
+  --ifm-color-primary-light: #ff858b;
+  --ifm-color-primary-lighter: #ffaeb2;
+  --ifm-color-primary-lightest: #ffd1d3;
   --ifm-code-font-size: 95%;
 }
 
 /* For readability concerns, you should choose a lighter palette in dark mode. */
 html[data-theme='dark'] {
-  --ifm-color-primary: #25c2a0;
-  --ifm-color-primary-dark: #21af90;
-  --ifm-color-primary-darker: #1fa588;
-  --ifm-color-primary-darkest: #1a8870;
-  --ifm-color-primary-light: #29d5b0;
-  --ifm-color-primary-lighter: #32d8b4;
-  --ifm-color-primary-lightest: #4fddbf;
+  --ifm-color-primary: #ff5760;
+  --ifm-color-primary-dark: #d23c43;
+  --ifm-color-primary-darker: #ae262d;
+  --ifm-color-primary-darkest: #540c0f;
+  --ifm-color-primary-light: #ff858b;
+  --ifm-color-primary-lighter: #ffaeb2;
+  --ifm-color-primary-lightest: #ffd1d3;
+}
+
+.footer {
+  background: #1a1a1a;
```

```suggestion
  background: #242526;
```

This change matches the dark mode navbar background color. I think it makes a nicer differentiation from the darkmode body background.